### PR TITLE
chore(Styleguide): Adds FlexBox story and configurable example [WEB-647]

### PR DESCRIPTION
**Pull request by codecaaron**
## Overview
Adds a missing story for this layout element.

### PR Checklist

- [ ] Related to Abstract designs:
- [x] Related to JIRA ticket: WEB-647
- [x] I have run this code to verify it works
- [ ] This PR includes unit tests for the code change

### Description

Adds a baseline story for this so at least we get a knob example and some things to play with.  Second pass might have some more description on how to use this, and more combinations, right now its fairly simple

Also removes some unused imports and typos from other stories.

### CODE REVIEW

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://5efe222f2a423802071fc558--codecademy-styleguide.netlify.app>https://5efe222f2a423802071fc558--codecademy-styleguide.netlify.app</a></p><p><a href=https://app.netlify.com/sites/codecademy-styleguide/deploys/5efe222f2a423802071fc558>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://5f0387b2ece51ce377e93c22--codecademy-styleguide.netlify.app>https://5f0387b2ece51ce377e93c22--codecademy-styleguide.netlify.app</a></p><p><a href=https://app.netlify.com/sites/codecademy-styleguide/deploys/5f0387b2ece51ce377e93c22>Deploy Logs</a></p>

File: packages/styleguide/stories/Core/Foundations/Layouts/FlexContainer.stories.mdx [hunk 1]
```diff
@@ -0,0 +1,56 @@
+import { Meta, Props, Preview, Story } from '@storybook/addon-docs/dist/blocks';
+import { StoryStatus } from '../../../../Blocks';
+import { Item, Container } from '@codecademy/gamut/src';
+import { Box, defaultGridProps } from './Elements';
+import { select, boolean, number } from '@storybook/addon-knobs';
+
+<Meta
+  title="Core|Foundations/Layouts/Container"
```
Comment (jakemhiller) [inline packages/styleguide/stories/Core/Foundations/Layouts/FlexContainer.stories.mdx:8]: overall lgtm, but i'm not sure if i'm a fan of how nested the layout stuff is, i feel like they could just be in atoms/molecules as components? it's not very discoverable through the nav at least

Comment (jakemhiller) [reply packages/styleguide/stories/Core/Foundations/Layouts/FlexContainer.stories.mdx:8]: not a blocker, we can get this in and discuss the nav separately. As an experiment I tested out what the nav would look like if we removed the nesting in the Core folder in this draft https://github.com/Codecademy/client-modules/pull/864

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://5f04db52497bc46f5486158a--codecademy-styleguide.netlify.app>https://5f04db52497bc46f5486158a--codecademy-styleguide.netlify.app</a></p><p><a href=https://app.netlify.com/sites/codecademy-styleguide/deploys/5f04db52497bc46f5486158a>Deploy Logs</a></p>

**Pull request closed without merging**